integration-cli/docker_cli_events_test.go

```diff
@@ -146,9 +146,21 @@ func TestEventsImageUntagDelete(t *testing.T) {
 func TestEventsImagePull(t *testing.T) {
 	since := time.Now().Unix()
 
-	defer deleteImages("hello-world")
+	defer setupRegistry(t)()
+	repoName := fmt.Sprintf("%v/dockercli/hello-world", privateRegistryURL)
+	defer deleteImages(repoName)
 
-	pullCmd := exec.Command(dockerBinary, "pull", "hello-world")
+	repo := fmt.Sprintf("%v:%v", repoName, "latest")
+	if out, _, err := runCommandWithOutput(exec.Command(dockerBinary, "tag", "busybox", repo)); err != nil {
+		t.Fatalf("Failed to tag image %v: error %v, output %q", repo, err, out)
+	}
+	defer deleteImages(repo)
```

Comment:
This `defer deleteImages(repo)` should be moved to the line after `repo` is created and `defer deleteImages(repoName)` can be removed.  `repoName` does not represent an image or tag and therefore does not need to be explicitly deleted, `repo` represents the tagged image to be removed.